modules/navier_stokes/src/bcs/INSFEFluidMomentumBC.C

```diff
@@ -132,32 +140,34 @@ INSFEFluidMomentumBC::computeQpOffDiagJacobian(unsigned int jvar)
   unsigned m = this->mapVarNumber(jvar);
   RealVectorValue vec_vel(_u_vel[_qp], _v_vel[_qp], _w_vel[_qp]);
   Real porosity = _has_porosity ? _porosity[_qp] : 1;
+  Real jac = 0;
 
-  // this is the jacobian w.r.t branch pressure
```

Comment:
it was nice having it before the switch, you could return earlier

Reply:
Updated in the latest commits.